tenant/store.go

```diff
@@ -38,10 +40,11 @@ func (tenantStore *tenantStore) Entities() []interface{} {
 }
 
 type Tenant struct {
-	Id       uint64 `sql:"AUTO_INCREMENT"`
-	Name     string
-	Segments []Segment
-	Seq      uint64
+	ID         uint64 `sql:"AUTO_INCREMENT"`
+	ExternalID string `sql:"not null;unique"`
```

Comment:
So there has to be a comment about which ID we should use in the REST requests generally (responses would have both). Is it - we have facility to look up our ID by External ID but our ID is primary - or we always use external?

This place in the code is probably is not a good place for such a comment as it would apply to all entities, but asking this question in the main tab doesn't keep the conversation straight so I'm adding this comment here.